src/components/NoTrajectoriesText/style.css

```diff
@@ -2,20 +2,17 @@
     margin: 1em;
 }
 
-.container p, .container h3 {
+.container div, .container h3 {
```

Comment:
I would give this a classname instead of targeting such a general tag type. this definitely opens you up to unintended application of style

Reply:
k i'll make a couple changes the component styling  make it more class focused